### DISCUSSION TITLE
Installing USB driver on Windows link dissabled

### DISCUSSION THREAD

**Discussion by pablocelotl** [bitcraze/discussions#811] [General]
I am trying to install the driver for the crazyradio PA. "First of all download Zadig from its website: http://zadig.akeo.ie/." This is the instruction on the webpage, but the link is currently not working. What could I do to install the driver? is there another link or method to install it?

Thank you very much.

**Comment by ataffanel** [comment]
The link seems to be working again. Zadig is the easiest to install the driver on Windows, I am not sure how to do it without it. It is also available form github: https://github.com/pbatard/libwdi/releases/latest.
